test/lightning_web/live/workflow_live/edit_test.exs

```diff
@@ -2110,22 +2108,22 @@ defmodule LightningWeb.WorkflowLive.EditTest do
       assert high_priority_view
              |> has_element?("#inspector-workflow-version", "latest")
 
-      dynamically_absorb_delay(
-        fn ->
-          !(low_priority_view
-            |> has_element?("#inspector-workflow-version", "latest"))
-        end,
-        sleep: 2
-      )
```

Comment:
@rorymckinley I'm trying something new here, you're not the first and won't be the last to deal with tricky timing issues on tests, the `retry` library has been in the back of my mind for a while, thought it would be worth taking it for a spin.